README.md

```diff
@@ -39,9 +41,9 @@ puppet module install puppetlabs-dsc
 
 ## Usage
 
-### Refresh Mode Must be Disabled
+### LCM RefreshMode Must be Disabled
```

Comment:
So I got word today that this is no longer a limitation based on the production preview bits.  @jpogran / @ferventcoder / @cowofevil want to verify that?